wgengine/magicsock/magicsock_unix_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build unix
```

Comment:
these lines shouldn't be required since the filename is _unix_test.go?

Reply:
I'm probably getting something wrong:

```
~ts/tailscale set-socket-buffer-size % GOOS=windows go test -c ./wgengine/magicsock 
# tailscale.com/wgengine/magicsock [tailscale.com/wgengine/magicsock.test]
wgengine/magicsock/magicsock_unix_test.go:30:37: cannot use int(fd) (value of type int) as type syscall.Handle in argument to syscall.GetsockoptInt
wgengine/magicsock/magicsock_unix_test.go:34:37: cannot use int(fd) (value of type int) as type syscall.Handle in argument to syscall.GetsockoptInt```

Reply:
https://pkg.go.dev/cmd/go#hdr-Build_constraints does not seem to document unix for file suffixes, only for tags. This seems to be in common with what appears in GOROOT.